### PR TITLE
Feature/15596 pupils not taking check remove

### PR DESCRIPTION
**Pull request by pris54**
Story 15596 ready for review.
I'm adding the latest added story for this feature: 16512
This PR depends on PR #216

### CODE REVIEW

File: admin/services/pupil.service.js [hunk 1]
```diff
@@ -25,6 +25,19 @@ const pupilService = {
     }
   },
   /**
+   * Fetch one pupil filtered by pupil id and school id
+   * @param pupilId
+   * @param schoolId
+   * @returns {Promise.<*>}
+   */
+  fetchOnePupil: async (pupilId, schoolId) => {
```
Comment (jon-shipley) [inline admin/services/pupil.service.js]: We should discuss naming at some point.  I prefer to see the service in the call e.g.
`pupilService.fetchOne(id)`

But I would guess from this name that you prefer 
`fetchOnePupil` which I think is less clear as it hides the service.

Comment (pris54) [reply admin/services/pupil.service.js]: @jon-shipley I prefer your suggestion, thanks. I'll have a refactoring PR next week with this and other improvements.

Comment (jon-shipley) [reply admin/services/pupil.service.js]: This should really be moved to pupil.data.service module.

Comment (pris54) [reply admin/services/pupil.service.js]: @jon-shipley  done in the refactoring branch

**Pull request merged**